client/src/components/GasStation.jsx

```diff
@@ -0,0 +1,37 @@
+
+import data from 'client/src/data/db.js';
+
+
+
+const GasStation = ({ name, uuid, e5 }) => {
+  const [gasStationDetailsOn, setGasStationDetailsOn] = useState(true);
+  const [gasStation, setGasStation] = useState({});
+  const handleDetails = () => {
+    setGasStationDetailsOn(changeState => !changeState);
+  };
+
+//   useEffect(() => {
+//     fetch('../data/db.js' + uuid).then(response =>
+//       response.json().then(data => setGasStation(data))
+//     );
+//   }, []);
+
+  return (
+    <>
+      {gasStationDetailsOn ? (
+        <li onClick={handleDetails}>
+          <h2>{name}</h2>
+        </li>
+      ) : (
+        <li onClick={handleDetails}>
+          <h2>{name}</h2>
+          <h2>{e5}</h2>
+        </li>
+      )}
```

Comment:
You could refactor this to:

```suggestion
        <li onClick={handleDetails}>
          <h2>{name}</h2>
          {gasStationDetailsOn && <h2>{e5}</h2>}
        </li>
```
and save some lines of code.

The two `h2`s are still not correct semantically speaking.